src/pk-main.c

```diff
@@ -173,6 +173,10 @@ main (int argc, char *argv[])
 	/* get values from the config file */
 	conf = g_key_file_new ();
 	conf_filename = pk_util_get_config_filename ();
+	if (conf_filename == NULL) {
+		g_print ("Cannot found config file.");
```

Comment:
Really really minor, but `Cannot found config file.` is weird English - maybe `Could not find config file.` or `Config file not found.` would be better here.
Or, IMHO even better and by using `g_printerr`:
```C
g_printerr ("Config file '%s' not found.", conf_filename);
```

Additionally, but unrelated to this PR: @hughsie, shouldn't we exit with a non-zero exit status if we encounter such initialization issues? At time, PK seems to always terminate with an exit code of zero.